frontend/src/app/app.module.ts

```diff
@@ -17,6 +20,10 @@ import { CampaignsComponent } from './campaigns/campaigns.component';
 import { GuideComponent } from './guide/guide.component';
 import { AnalyticsComponent } from './analytics/analytics.component';
 import { IntroComponent } from './intro/intro.component';
+import {MatTableModule} from '@angular/material/table';
```

Comment:
Add some spacing before and after `MatTableModule`.

Reply:
Sure